docs/sphinx/source/reference/sql_commands/DQL/Operators/Comparison.rst

```diff
@@ -2,11 +2,101 @@
 Comparison Operators
 ====================
 
-
 .. _comparison-operators:
 
-TODO: flesh out doc
+Standard Comparison Operators
+==============================
+
+We support standard comparison operators: ``>``, ``>=``, ``=``, ``<``, ``<=``, ``!=``, ``<>``
+
+If either the left or right operand is NULL, the comparison result is NULL.
+
+.. code-block:: sql
+
+    SELECT * FROM products WHERE price > 100
+    SELECT * FROM products WHERE price <= 50
+    SELECT * FROM products WHERE name = 'Widget'
+    SELECT * FROM products WHERE stock != 0
+
+IS DISTINCT FROM
```

Comment:
I see that the `IS DISTINCT FROM` operator is currently placed under `Comparison Operators` as well as described in its own page. However, the latter is more elaborate. Is this intentional in the sense that there should be a small summary of the operator in the `Comparison Operators` page?